# HG changeset patch
# Parent d37b4be9a268087980e52156a037411fec3429e8
Build: Disable several whole directories

On the chopping block include:
 -examples
 -blktap2
 -remus
 -gdbsx
 -Ocaml tests (they dont build correctly)

diff -r d37b4be9a268 tools/Makefile
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -7,7 +7,6 @@ SUBDIRS-y += libxc
 SUBDIRS-$(FLASK_ENABLE) += flask
 SUBDIRS-y += xenstore
 SUBDIRS-y += misc
-SUBDIRS-y += examples
 SUBDIRS-y += hotplug
 SUBDIRS-y += xentrace
 SUBDIRS-$(CONFIG_XCUTILS) += xcutils
@@ -19,8 +18,6 @@ SUBDIRS-$(CONFIG_Linux) += memshr
 ifeq ($(CONFIG_X86)$(CONFIG_Linux),yy)
 SUBDIRS-$(CONFIG_BLKTAP1) += blktap
 endif
-SUBDIRS-$(CONFIG_Linux) += blktap2
-SUBDIRS-$(CONFIG_NetBSD) += blktap2
 SUBDIRS-$(CONFIG_NetBSD) += xenbackendd
 SUBDIRS-y += libfsimage
 SUBDIRS-$(LIBXENAPI_BINDINGS) += libxen
@@ -33,9 +30,7 @@ endif
 
 SUBDIRS-y += xenpmd
 SUBDIRS-y += libxl
-SUBDIRS-y += remus
 SUBDIRS-$(CONFIG_X86) += xenpaging
-SUBDIRS-$(CONFIG_X86) += debugger/gdbsx
 SUBDIRS-$(CONFIG_X86) += debugger/kdd
 SUBDIRS-$(CONFIG_TESTS) += tests
 
diff -r d37b4be9a268 tools/ocaml/Makefile
--- a/tools/ocaml/Makefile
+++ b/tools/ocaml/Makefile
@@ -1,7 +1,7 @@
 XEN_ROOT = $(CURDIR)/../..
 include $(XEN_ROOT)/tools/Rules.mk
 
-SUBDIRS_PROGRAMS = xenstored test
+SUBDIRS_PROGRAMS = xenstored
 
 SUBDIRS = libs $(SUBDIRS_PROGRAMS)
 
